lib/paperclip/schema.rb

```diff
@@ -40,7 +40,11 @@ def remove_attachment(table_name, *attachment_names)
         attachment_names.each do |attachment_name|
           COLUMNS.each_pair do |column_name, column_type|
             column_options = options.merge(options[column_name.to_sym] || {})
-            remove_column(table_name, "#{attachment_name}_#{column_name}", column_type, column_options)
+            if ActiveRecord::VERSION::MAJOR >= 4
+              remove_column(table_name, "#{attachment_name}_#{column_name}", column_type, column_options)
```

Comment:
We don't need that, see 127e8797fb3c6daa2dbcd6763778d58b6645777f and 6ea7c26512434a93b598602fd6e959452a97969b. Will remove.